scripts/local_testnet_pos/vars.env

```
@@ -0,0 +1,50 @@
+# Base directory for all data files.
+DATADIR=~/.lighthouse/testnet-with-geth
```

```suggestion
DATADIR=~/.lighthouse/local-execution-testnet
```

I'd prefer a name more similar to the existing `~/.lighthouse/local-testnet` name